--- RBio/Makefile.orig	2011-12-07 23:20:05.000000000 +0100
+++ RBio/Makefile	2012-01-08 14:28:27.806553147 +0100
@@ -6,7 +6,7 @@
 
 default: all
 
-include ../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
 # Compile the C-callable libraries and the Demo programs.
 all:
--- RBio/Lib/Makefile.orig	2011-12-08 18:41:19.000000000 +0100
+++ RBio/Lib/Makefile	2012-01-08 14:38:26.373239855 +0100
@@ -6,7 +6,7 @@
 
 ccode: all
 
-include ../../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
 C = $(CC) $(CF)
 
@@ -51,7 +51,7 @@
 
 $(OBJ): $(INC)
 
-I = -I../Include -I../../SuiteSparse_config
+I = -I../Include
 
 #-------------------------------------------------------------------------------
 
--- RBio/Demo/Makefile.orig	2012-05-27 03:17:26.000000000 +0200
+++ RBio/Demo/Makefile	2012-06-27 21:42:48.921405153 +0200
@@ -4,9 +4,9 @@
 
 default: all
 
-include ../../SuiteSparse_config/SuiteSparse_config.mk
+include /usr/share/misc/SuiteSparse_config.mk
 
-CLIB = ../Lib/librbio.a ../../SuiteSparse_config/libsuitesparseconfig.a $(LIB)
+CLIB = ../Lib/librbio.a -lsuitesparseconfig $(LIB)
 
 C = $(CC) $(CF)
 
@@ -26,11 +26,10 @@
 
 #-------------------------------------------------------------------------------
 library:
-	( cd ../../SuiteSparse_config ; $(MAKE) )
 	( cd ../Lib ; $(MAKE) )
 #-------------------------------------------------------------------------------
 
-I = -I../Include -I../../SuiteSparse_config
+I = -I../Include
 
 RBdemo: library RBdemo.c
 	$(C) -o RBdemo $(I) RBdemo.c $(CLIB)
